ldms/python/ldmsd/ldmsd_communicator.py

```diff
@@ -204,17 +204,6 @@
                       'auth_add': {'req_attr': ['name', 'plugin'], 'opt_attr': []},
                       }
 
-def check_offset(interval_us, offset_us=None):
```

Comment:
Why did we decide that this check was no longer necessary?

Reply:
- The main reason is that LDMSD also checks the offset value.
- With the patch, LDMSD converts interval strings to microseconds, so we don't have to duplicate the code in ldmsctl, ldmsd_controller, and the config file parser.
- If we want to keep the check_offset(), I need to convert interval strings to microseconds before passing the values to check_offset(). I will probably move the conversion and the check logic out from LDMSD and put them in ldmsctl and config file parser, similar to ldmsd_controller.